src/progressbar.rs

```diff
@@ -0,0 +1,5 @@
+use indicatif::ProgressBar;
```

Comment:
how do you feel about `extern crate`s that are not in the root module? I would probably put the `extern crate indicatif;` here

Reply:
not generally a huge fan :/